sdk/core/azure-core/src/main/java/com/azure/core/util/ServiceVersion.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+/**
+ * A generic interface for sending HTTP requests using the provided service version.
+ */
+public interface ServiceVersion {
+
+    /**
+     * Gets the string representation of the {@link ServiceVersion}
+     *
+     * @return the string representation of the {@link ServiceVersion}
+     */
+    String getVersionString();
```

Comment:
Do we need to name it as `getVersionString` instead of just `getVersion` as it returns `String` anyway?

Reply:
I agree.